Modules/_operator.c

```diff
@@ -698,6 +698,20 @@ _operator_index(PyObject *module, PyObject *a)
     return PyNumber_Index(a);
 }
 
+/*[clinic input]
+_operator.is_none = _operator.neg
+
+Same as a is None.
```

Comment:
```suggestion
Same as 'a is None'.
```

Reply:
If we made this change, it would make it different to all other docstrings in this module

Reply:
Maybe not then :)